source/ext/patches/fluentd/in_syslog.patch

```diff
@@ -0,0 +1,20 @@
+--- ../source/ext/fluentd/lib/fluent/plugin/in_syslog.rb	2019-04-15 15:49:46.112195768 -0700
++++ ../source/ext/fluentd/lib/fluent/plugin/in_syslog.rb.new	2019-04-15 16:04:08.530462598 -0700
+@@ -110,6 +110,8 @@
+     # 13 is the default value of rsyslog and syslog-ng
+     desc 'The default PRI value (0 - 191 are available)'
+     config_param :default_priority, :integer, default: 13
++    desc 'The delimiter value "\n"'
++    config_param :delimiter, :string, default: "\n" # syslog family add "\n" to each message
```

Comment:
"syslog family" do you mean "syslog facility"?

Reply:
@hestolz syslog family mean: rsyslog, syslog-ng and others

Reply:
done